src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  place-items: center;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
```

Comment:
Verify that for a minimum width of 768px, the layout corresponds to the design specifications. Adjust the repeat function and column width if needed.